### PR TITLE
Use dockerhub image instead of building it

### PR DESCRIPTION
**Pull request by garyservin**
@ernestmc or @adamantivm

### CODE REVIEW

File: do_docker.sh [hunk 1]
```diff
@@ -73,21 +73,21 @@ echo
 # Requires docker 1.3+
 cmd_exists docker || die 'docker was not found'
 
-echo -e '\e[34mCreting docker image.\e[39m'
-sudo docker build -t rosndk ./docker
+echo -e '\e[34mPull base docker image.\e[39m'
```
Comment (adamantivm) [inline do_docker.sh]: nit: Pull --> Pulling ... from docker hub registry

**Comment by adamantivm** [comment]
looks good other than minor comment nit. It would be nice to have someone else test it before merging.

**Comment by garyservin** [comment]
I've tested it on my existing workspace, I'll try it with a fresh workspace.

**Comment by adamantivm** [comment]
I think it would be much better if you get someone else to try it, for better variability control

**Comment by ernestmc** [comment]
I'll test it.

**Comment by ernestmc** [comment]
+1
Looks good to merge.

**Pull request merged**